File: app/src/main/java/com/callmangement/EHR/models/FeedbackDetailByFpsRoot.java
package com.callmangement.EHR.models;

public class FeedbackDetailByFpsRoot {
    public FeedbackDetailByFpsData Data;
    public String message;
    public String status;


    public FeedbackDetailByFpsRoot(FeedbackDetailByFpsData data, String message, String status) {
        Data = data;
        this.message = message;
        this.status = status;
    }


    public FeedbackDetailByFpsData getData() {
        return Data;
    }

    public void setData(FeedbackDetailByFpsData data) {
        Data = data;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/FeedbackDetailByFpsRoot.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName

class FeedbackDetailByFpsRoot(
    @SerializedName("Data")
    var data: FeedbackDetailByFpsData,
    @JvmField var message: String,
    @JvmField var status: String
)
